svgr-template.js

```diff
@@ -84,7 +89,7 @@ const ${variables.componentName} = forwardRef((${variables.props}, ref) => {
     other.width = isNaN(width) ? "100%" : width
   }
 
-  const fill = other.fill ?? theme.color?.icon[color]
+  let fill = other.fill ?? theme.color?.icon[color]
```

Comment:
maybe a comment here - easy to miss the thing above. smart though.

Reply:
yeah def ^